README.md

```diff
@@ -149,7 +152,6 @@ The following list contains proposals that have been under review and for which
 - [Extended Local Chain Sync Protocol](https://github.com/cardano-foundation/CIPs/pull/375)
 - [Tiered Pricing Protocol](https://github.com/cardano-foundation/CIPs/pull/381)
 - [Properly burning NFTs/tokens](https://github.com/cardano-foundation/CIPs/pull/392)
-- [NFT Metadata extension tag](https://github.com/cardano-foundation/CIPs/pull/343)
 
 <p align="right"><i>Last updated on 2023-03-15</i></p>
```

Comment:
```suggestion
<p align="right"><i>Last updated on 2023-05-24</i></p>
```
Match today's date? 😎

Reply:
ugh, I knew I missed something; thanks 👍

Reply:
and there is another date on line 140!

Reply:
yes, I've updated that one too, and since we effectively have 4 lists which may be updated at any time the list of candidates (with `?`'s) was missing its own date stamp.  All of them should have it now, and all dates are updated except for the `CPS` list which hasn't changed.  Should be good to go. 🤪